blocked-edges/4.14.47-LabeledMachineConfigAndContainerRuntimeConfigBlocksMCO.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.47
+from: .*
+url: https://issues.redhat.com/browse/MCO-1585
+name: LabeledMachineConfigAndContainerRuntimeConfigBlocksMCO
+message: |-
+  KubeletConfig/ContainerRuntimeConfig cannot be applied and 
+  the cluster upgrade will not be complete because MachineConfigOperator 
+  is stuck with the pending state if all of the following conditions are satisfied:
+
+  * There exists a KubeletConfig/ContainerRuntimeConfig whose has a label in its metadata matching a label on a MachineConfigPool, and
+
+  * The KubeletConfig/ContainerRuntimeConfig whose generation in its metadata is greater than 0 or/and 
```

Comment:
This dual-condition seems complicated.  Maybe we should simplify the message to be something like:

```yaml
message: Some clusters with some KubeletConfigs or ContainerRuntimeConfigs will have issues updating MachineConfigPools.
```

and then folks who are concerned can follow the link to the impact-statement, where the details get covered.  With fixes shipping in all supported, impacted 4.y streams, I doubt there are that many folks who will want to dig into "am I exposed?" vs. just updating to a patched release, which is why I think we can avoid trying to push "how to manually check and see if I'm exposed" out into each cluster.

Reply:
> just updating to a patched release

That is what I would do whenever the fix is available and upgradable.
The safest way is to bypass any risk. :)

Reply:
Done.